src/wasm-lib/kcl/src/std/sketch.rs

```diff
@@ -124,17 +131,18 @@ async fn inner_line_to(
 
     args.batch_modeling_cmd(
         id,
-        ModelingCmd::ExtendPath {
-            path: sketch_group.id,
-            segment: kittycad::types::PathSegment::Line {
-                end: Point3D {
+        ModelingCmd::from(mcmd::ExtendPath {
+            path: sketch_group.id.into(),
+            segment: PathSegment::Line {
+                end: KPoint3d {
```

Comment:
I've aliased this to `KPoint3d` instead of `Point3d`, because `Point3d` is already defined in this codebase. 

The reason we're using this codebase's symbol is because `modeling-api` doesn't (yet) support ts-rs.